app/src/main/java/org/simple/clinic/storage/migrations/Migration_62.kt

```diff
@@ -0,0 +1,51 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.simple.clinic.storage.inTransaction
+import javax.inject.Inject
+
+@Suppress("ClassName")
+class Migration_62 @Inject constructor() : Migration(61, 62) {
+
+  override fun migrate(database: SupportSQLiteDatabase) {
+    database.inTransaction {
+      execSQL("""
+        ALTER TABLE "LoggedInUser" RENAME TO "LoggedInUser_old"
+      """)
+
+      execSQL("""
+        CREATE TABLE IF NOT EXISTS "LoggedInUser" (
+            "uuid" TEXT NOT NULL, "fullName" TEXT NOT NULL, "phoneNumber" TEXT NOT NULL, 
+            "pinDigest" TEXT NOT NULL, "status" TEXT NOT NULL, 
+            "createdAt" TEXT NOT NULL, "updatedAt" TEXT NOT NULL, 
+            "loggedInStatus" TEXT NOT NULL, 
+            "registrationFacilityUuid" TEXT NOT NULL, "currentFacilityUuid" TEXT NOT NULL, 
+            PRIMARY KEY("uuid"), 
+            FOREIGN KEY("registrationFacilityUuid") REFERENCES "Facility"("uuid") ON UPDATE NO ACTION ON DELETE NO ACTION , 
+            FOREIGN KEY("currentFacilityUuid") REFERENCES "Facility"("uuid") ON UPDATE NO ACTION ON DELETE NO ACTION)
+      """)
+
+      execSQL(""" CREATE INDEX "index_LoggedInUser_registrationFacilityUuid" ON "LoggedInUser" ("registrationFacilityUuid") """)
+      execSQL(""" CREATE INDEX "index_LoggedInUser_currentFacilityUuid" ON "LoggedInUser" ("currentFacilityUuid") """)
+
+      execSQL("""
+        INSERT INTO "LoggedInUser" (
+            "uuid", "fullName", "phoneNumber",
+            "pinDigest", "status", "createdAt",
+            "updatedAt", "loggedInStatus",
+            "registrationFacilityUuid", "currentFacilityUuid"
+        )
+        SELECT U."uuid", U."fullName", U."phoneNumber",
+            U."pinDigest", U."status", U."createdAt",
+            U."updatedAt", U."loggedInStatus",
+            FM."facilityUuid", FM."facilityUuid"
```

Comment:
I thought about this, and I made this decision as part of the migration. Since we fetch the user information each time the app opens, this registration facility will get set correctly the next time the user API fetches.

Reply:
If the fetch overrides the existing information, then it is alright 👍

Reply:
Yup, I tested and verified this.

Reply:
I'll leave a comment in the migration.